openstf/model/regressors/custom_regressor.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+from typing import List, Type
+from abc import abstractmethod
+from importlib import import_module
+import pandas as pd
+import inspect
+from .regressor import OpenstfRegressor
```

Comment:
Preferably we use full paths for package/module imports and not relative paths. It's more readable and works better

Reply:
Same for row below (line 11)